src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -172,34 +168,4 @@ private static DetailNode parseFile(File file) throws IOException {
             System.getProperty("file.encoding", "UTF-8"));
         return parseJavadocAsDetailNode(text.getFullText().toString());
     }
-
-    /**
-     * Creates DetailAST block comment to pass it to the Javadoc parser.
-     * @param content comment content.
-     * @return DetailAST block comment
-     */
-    private static DetailAST createFakeBlockComment(String content) {
```

Comment:
> surviving mutation is same between master pit-test reports and patch pit test reports and all the new mutations are being killed
> 100% 211/212

Issue is utility rounds up. 211/212 does not equal 100%. It should have calculated it as 99%.
Decrease in config is ok in this situation, IMO.

> 4643#discussion

@ps-sp Thanks for pointing this out, I didn't see my name referenced.
Yes, this looks like the same as the other issue, but with the addition of moving the method to a different class.

I am right more ok with this change than previous because you made it a utility method and it sort of looks like one. I dislike the fact that we need it as a utility method with only one production code call to it.
If we are going to keep this, can another place call this method. What about `TreeWalker#createBlockCommentNode`?